src/shared/components/palette/address/write-address-field.html

```diff
@@ -4,20 +4,20 @@
     <h2 class="heading-h2">{{caseField | ccdFieldLabel }}</h2>
 
     <div class="form-group bottom-30" id="postcodeLookup" [ngClass]="{'form-group-error': missingPostcode}" *ngIf="!isExpanded">
-      <label for="postcodeInput">
+      <label [for]="id() + '_postcodeInput'">
         <span class="form-label">Enter a UK postcode</span>
       </label>
-        <span class="error-message" *ngIf="missingPostcode">Enter the Postcode</span>
-        <input type="text" id="postcodeInput" name="postcode" class="form-control postcodeinput" [formControl]="postcode">
+      <span class="error-message" *ngIf="missingPostcode">Enter the Postcode</span>
+      <input type="text" [id]="id() + '_postcodeInput'" name="postcode" class="form-control postcodeinput" [formControl]="postcode">
```

Comment:
there's lot of duplication in the creation of the id. Rather that manually concatenating everywhere `id()+`
we should create a method in the component called `createId(string)` and call the method here:
createId(postcodeInput), createId(addressList)

Reply:
Done.